Packs/AzureDevOps/Integrations/AzureDevOps/README.md

```diff
@@ -1,5 +1,6 @@
 # Azure DevOps
 Manage Git repositories in Azure DevOps Services. Integration capabilities include retrieving, creating, and updating pull requests. Run pipelines and retrieve git information.
+** Note: This is a beta Integration, which lets you implement and test pre-release software. Since the integration is beta, it might contain bugs. Updates to the integration during the beta phase might include non-backward compatible features. We appreciate your feedback on the quality and usability of the integration to help us identify issues, fix them, and continually improve.
```

Comment:
@DanilVil Did you ask for this?

Reply:
@ShahafBenYakir Yes it was requested in the issue: [28923](https://github.com/demisto/etc/issues/28923)